query/core/core_network_security_list_restrict_ingress_rdp_all.sql

```diff
@@ -31,6 +31,7 @@ select
   end as reason,
   -- Additional Dimensions
   region,
+  nsg.compartment_id,
   coalesce(c.title, 'root') as title
```

Comment:
I'm confused on what our standard set of additional dimensions are across queries in this mod. Here we have region, compartment_id, and a compartment name as `title`, while in others we just have region and compartment_id, and in other queries, like https://github.com/turbot/steampipe-mod-oci-compliance/blob/a43e4dff6b243d26ecb020312c56f1317b5e1a9d/query/kms/kms_cmk_rotation_365.sql#L35, we have region and the compartment name as `compartment`.